lib/opFns.js

```diff
@@ -122,22 +122,22 @@ module.exports = {
   },
   // 0x10 range - bit ops
   LT: function (a, b, runState) {
-    return new BN(a.cmp(b) === -1)
+    return new BN(a.cmp(b) === -1 ? 1 : 0)
   },
   GT: function (a, b, runState) {
-    return new BN(a.cmp(b) === 1)
+    return new BN(a.cmp(b) === 1 ? 1 : 0)
   },
   SLT: function (a, b, runState) {
-    return new BN(a.fromTwos(256).cmp(b.fromTwos(256)) === -1)
+    return new BN(a.fromTwos(256).cmp(b.fromTwos(256)) === -1 ? 1 : 0)
   },
   SGT: function (a, b, runState) {
-    return new BN(a.fromTwos(256).cmp(b.fromTwos(256)) === 1)
+    return new BN(a.fromTwos(256).cmp(b.fromTwos(256)) === 1 ? 1 : 0)
   },
   EQ: function (a, b, runState) {
-    return new BN(a.cmp(b) === 0)
+    return new BN(a.cmp(b) === 0 ? 1 : 0)
```

Comment:
Oh this code was done way before the higher level helpers were added to bn.js. We could use `a.eq(b)` here and above.

Reply:
Created a new PR for this.